Mail/Views/Thread/Message/MessageBodyView.swift

```diff
@@ -108,23 +105,26 @@ struct MessageBodyView: View {
         }
     }
 
-    private func loadBody(blockRemoteContent: Bool) {
+    private func loadBody(presentableBody: PresentableBody, blockRemoteContent: Bool) {
+        let uid = messageUid
+
         Task {
             let loadResult = try await model.loadBody(
                 presentableBody: presentableBody,
                 blockRemoteContent: blockRemoteContent,
-                messageUid: messageUid
+                messageUid: uid
             )
 
-            displayContentBlockedActionView = (loadResult == .remoteContentBlocked)
+            Task { @MainActor in
+                displayContentBlockedActionView = (loadResult == .remoteContentBlocked)
```

Comment:
```suggestion
                displayContentBlockedActionView = loadResult == .remoteContentBlocked
```
We can use a more swifty syntax :)